waku/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -434,6 +434,16 @@ proc getInt(s: PostgresDriver,
 
   return ok(retInt)
 
+method getDatabasesSize*(s: PostgresDriver):
+                         Future[ArchiveDriverResult[int64]] {.async.} =
+
+  let intRes = await s.getInt("SELECT pg_database_size(current_database())")
+  if intRes.isErr():
+    return err("error in getMessagesCount: " & intRes.error)
```

Comment:
```suggestion
  let int = (await s.getInt("SELECT pg_database_size(current_database())")).valueOr:
    return err("error in getMessagesCount: " & error)
```

Reply:
thanks for the input SP, such refactoring makes sense but then breaks the general code level uniformity, if you see in the original file they all follow such error handling. every time I look at it I am like "ah it breaks the flow", should we tell someone to refactor this across code base?